src/SFA.DAS.PR.Jobs.UnitTests/SFA.DAS.PR.Jobs.UnitTests.csproj

```diff
@@ -29,9 +29,17 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="17.1.94" />
```

Comment:
We are not using xunit, so why this?

Reply:
**SFA.DAS.Testing.AutoFixture** is used to use the **CreateMany** extension on Fixture

var providers = fixture
    .Build<Provider>()
    .CreateMany(p => p.Ukprn, ValidUkprns)
    .ToList();

Reply:
Sorry I think it may have moved or removed, but there was a reference to  XUnit

Reply:
I can see `<PackageReference Include="xunit.extensibility.core" Version="2.9.0" />` is still referenced

Reply:
@DevenKShah I've removed that package reference there now.